web/src/views/DandisetLandingView/ListingComponent.vue

```diff
@@ -1,72 +1,22 @@
 <template>
-  <div>
-    <template v-if="schema.type === 'array'">
-      <template v-if="schema.items.type === 'object'">
-        <v-expansion-panels>
-          <v-expansion-panel
-            v-for="item in data"
-            :key="item[primaryKey]"
-          >
-            <!-- item is an object -->
-            <v-expansion-panel-header>{{ item[primaryKey] }}</v-expansion-panel-header>
-            <v-expansion-panel-content>
-              <v-list>
-                <v-list-item
-                  v-for="key in Object.keys(item).sort()"
-                  :key="key"
-                >
-                  <!-- value's type matches that specified at schema.items.properties[key].type -->
-                  <ListingComponent
-                    :schema="schema.items.properties[key]"
-                    :data="item[key]"
-                  />
-                </v-list-item>
-              </v-list>
-            </v-expansion-panel-content>
-          </v-expansion-panel>
-        </v-expansion-panels>
-      </template>
-      <template v-else-if="schema.items.type === 'array'">
-        <!-- May not work. Not tested in a real scenario. -->
-        <ListingComponent
-          v-for="item in data"
-          :key="item"
-          :schema="schema.items"
-          :data="item"
-        />
-      </template>
-      <template v-else>
-        <template v-if="!root">
-          <b>{{ schema.title }}</b>:
-        </template>
-        {{ data.join(', ') }}
-      </template>
-    </template>
-    <template v-else-if="schema.type === 'object'">
-      <v-list dense>
-        <v-list-item
-          v-for="key in Object.keys(data).sort()"
-          :key="key"
-          dense
-        >
-          <ListingComponent
-            :schema="schema.properties[key]"
-            :data="data[key]"
-          />
-        </v-list-item>
-      </v-list>
-    </template>
-    <template v-else>
-      <!-- Base Case -->
-      <template v-if="!root">
-        <b>{{ schema.title }}</b>:
-      </template>
-      {{ data }}
-    </template>
-  </div>
+  <component
```

Comment:
I can't figure out where this `component` element comes from. Am I missing something obvious?

Reply:
This is a native `Vue` thing. `component` is what's used if you want to render a dynamic component, with the exact component you want to render being specific by the `is` prop. Docs: https://vuejs.org/v2/guide/components.html#Dynamic-Components